tanjun/conversion.py

```diff
@@ -1186,6 +1251,68 @@ def parse(value: _ArgumentT, /) -> list[hikari.Snowflake]:
     List of the user IDs in the string.
 """
 
+_MESSAGE_LINK_REGEX = re.compile(r"(\d+|@me)/(\d+)/(\d+)")
+
+
+def parse_message_id(
+    value: _ArgumentT,
+    /,
+    *,
+    message: str = "No valid message link or ID found",
+) -> typing.Tuple[typing.Optional[hikari.Snowflake], hikari.Snowflake]:
```

Comment:
```suggestion
) -> tuple[typing.Optional[hikari.Snowflake], hikari.Snowflake]:
```

Reply:
Oh right, forgot this library only targets 3.9+ 😅